packages/wallets/.eslintrc.js

```diff
@@ -64,4 +66,9 @@ module.exports = {
         'typescript-sort-keys/interface': 'error',
         'typescript-sort-keys/string-enum': 'error',
     },
+    settings: {
+        'import/resolver': {
+            webpack: { config: webpackConfig({}) },
+        },
+    },
```

Comment:
@markwylde-deriv could you please explain how this part works?

Reply:
It's used by ESLint to resolve module paths according to the Webpack bundling rules. It's what [dtrader does](https://github.com/binary-com/deriv-app/blob/master/packages/trader/.eslintrc.js).